build/toolchain/android/BUILD.gn

```diff
@@ -94,6 +94,30 @@ android_clang_toolchain("android_clang_x86") {
   }
 }
 
+android_clang_toolchain("cobalt_android_clang_x86") {
+  toolchain_args = {
+    current_cpu = "x86"
+
+    # We lack the libclang_rt.profile library for x86 and x86_64, so we cannot
+    # link any binaries that are generated with coverage instrumentation.
+    # Therefore we need to turn off 'use_clang_coverage' for this toolchain.
+    # TODO(crbug.com/865376)
+    use_clang_coverage = false
+
+    # This turns off all of the LaCrOS-specific flags. A LaCrOS related build
+    # may use |ash_clang_x64| or |lacros_clang_x64| toolchain, which are
+    # chromeos toolchains, to build Ash-Chrome or Lacros-Chrome in a
+    # subdirectory, and because chromeos toolchain uses android toolchain, which
+    # eventually resulted in that android toolchains being used inside a LaCrOS
+    # build.
+    also_build_ash_chrome = false
+    also_build_lacros_chrome = false
+    chromeos_is_browser_only = false
+    ozone_platform = ""
+    ozone_platform_wayland = false
+  }
+}
+
 android_clang_toolchain("android_clang_arm") {
```

Comment:
Had to modify android_clang_arm because of this error:   https://b.corp.google.com/issues/371241293#comment25